scripts/build-crossdock.sh

```diff
@@ -10,12 +10,18 @@ make build-and-run-crossdock
 # Only push images to dockerhub/quay.io for master branch
 if [[ "$BRANCH" == "master" ]]; then
   echo 'upload images to dockerhub/quay.io'
+  make build-crossdock-binary GOOS=linux GOARCH=amd64
+  make build-crossdock-binary GOOS=linux GOARCH=s390x
+  PLATFORMS="linux/amd64,linux/s390x"
+  REPO=jaegertracing/test-driver
+  IMAGE_TAGS=$(bash scripts/compute-tags.sh $REPO)
+  IMAGE_TAGS="${IMAGE_TAGS} --tag docker.io/${REPO}:${COMMIT} --tag quay.io/${REPO}:${COMMIT}"
+  bash scripts/docker-login.sh
+  docker buildx build --push \
```

Comment:
You're right, we do not need to make crossdock images multi-arch. I'll revert this script to use `docker build`.

Reply:
Hi @yurishkuro , since we've removed script `upload-to-registry.sh` for eliminating redundancy, I think here we could set `platforms` to `linux/amd64` and keep using `docker buildx` to upload images for code simplicity. I've updated script `build-crossdock.sh`. Please let me know what you think, thanks!

Reply:
What's the penalty in terms of clock time? buildx is slower than regular builds, isn't it?

Reply:
Yeah, `docker buildx` might be slower than regular build, but it builds the images concurrently and would bring the benefits of easy maintenance of multi-arch images.

Reply:
How much slower? If we talking about a couple of seconds slower in total, it's fine. If it adds 5 minutes to the build, it's not acceptable.

Reply:
I made a test from my end. The workflow `ci-all-in-one-build`, `ci-docker-build` and `ci-hodrod` will take 10m 47s, 9m 21s and 2m 2s when using regular build for image building respectively. When using `docker buildx` to build multi-arch images, these workflows will take 10m 14s, 12m 46s and 2m 19s. 

The workflow `ci-docker-build` might be slower when building multi-arch images since it has 13 different images to build. Maybe we could reduce the frequency of this workflow or divide it into smaller parallel tasks to mitigate the time penalty.

Reply:
I think the penalty isn't high enough right now to block this PR. We should certainly consider rethinking our CI, to avoid building all the things all the time if we don't need them for the tests. We can then have a nightly build to publish the snapshot images.

Reply:
Sure, I've opened an issue #3113 for tracking. Thanks!